index.html

```diff
@@ -8,20 +8,33 @@
     </head>
     <body class="overflow-hidden">
         <header>
-            <nav class="flex w-full h-16 bg-black text-white items-center">
+            <nav class="flex w-full h-[8vh] bg-black text-white items-center bg-gradient-to-t from-gray-800 via-gray-900 to-black">
                 <a href="#" class="flex items-center justify-start"><img src="assets/logo.png" alt="A.K.A" width="64" height="100"></a>
                 <a href="#" class="grid ml-6">Home</a>
                 <a href="#" class="grid ml-6">About</a>
                 <a href="#" class="grid ml-6">Content</a>
             </nav>
         </header>
         <main class="bg-hero bg-cover">
-            <div class="flex h-screen flex-col items-center justify-center">
+            <div class="flex min-h-[92vh] flex-col items-center justify-center">
```

Comment:
sorry that i didnt spot this change before, but i believe the div can be merged with the main class perfectly fine, hence it is being used redundantly.

also as stated above, i think that the dynamic screen sizing is not the best option. Instead of this screen sizing, to achieve a fullscreen background, we could use the `grow` attribute for flexbox and add `flex flex-col` to the body.

```suggestion
            <main class="bg-hero bg-cover grow flex flex-col items-center justify-center">
```

Reply:
Made a commit d336443b54664d0e56469a06a8105750746b211b.